src/main/java/org/mitre/synthea/world/agents/Provider.java

```diff
@@ -49,15 +55,26 @@
   public String type;
   public String ownership;
   public int quality;
+  public static String numClinicians;
+  public ArrayList<Clinician> clinicians;
   private DirectPosition2D coordinates;
-  private ArrayList<String> servicesProvided;
+  public ArrayList<String> servicesProvided;
+  private Map<String,Integer> specialtyInfo;
```

Comment:
Are the specialtyInfo, clinicians, and numClinicians fields used anywhere? If they are just written to and never used they should be removed.